.travis.yml

```diff
@@ -14,15 +14,19 @@
 # limitations under the License.
 
 language: java
-
-jdk:
-  - openjdk8
-  - openjdk11
-  - openjdk12
-  - openjdk13
-  - openjdk14  
-  - openjdk-ea
-
+cache:
+  directories:
+    - $HOME/.m2

```

Comment:
Oh, that sounds smart! I'm using something similar with GH actions but for JS and Ruby (caching `yarn.lock` and `Gemfile.lock`). And this little trick has sped up significantly my builds. :rocket: